### PR TITLE
Fixing import in manual_test.py broken after #807

### PR DESCRIPTION
**Pull request by MollieBakal**
Minor fix for import no longer found. Quick question--given the recent work converting module names to abide by snake_case, should BrowserParams be browser_params?

### CODE REVIEW

**Comment by MollieBakal** [comment]
Just noticed more changes which need to be made in the area--sorry. Also realized I forgot to add in pre-commit, which has since been rectified.

**Comment by vringar** [comment]
Thank you for this PR!
As BrowserParams is a class and not a module I think CamelCase is the appropriate formatting.

**Comment by MollieBakal** [comment]
You're right--I just looked at PEP8, which I hadn't before. Interesting :).

File: test/manual_test.py [hunk 1]
```diff
@@ -127,12 +127,12 @@ def cleanup_server():
...
 
-        browser_params = load_default_params()[1][0]
+        browser_params = BrowserParams()
         if browser_params_file is not None:
             with open(browser_params_file, "r") as f:
-                browser_params.update(json.loads(f.read()))
+                browser_params.from_json(f.read())
         js_request = browser_params.js_instrument_settings
-        js_request_as_string = jsi.convert_browser_params_to_js_string(js_request)
+        js_request_as_string = json.dumps(js_request)
```
Comment (vringar) [inline test/manual_test.py:135]: Please revert the change in this line. The js_request isn't a valid json unfortunately, so we need to call out to that method.

**Review by vringar — CHANGES_REQUESTED** [review]

File: test/manual_test.py (hunk 1 above)
Comment (MollieBakal) [reply test/manual_test.py:135]: Sorry about that--but I don't see the function `convert_browser_params_to_js_string`, and that was throwing an error at me. Is the correct function `clean_js_instrumentation_settings`?

Comment (vringar) [reply test/manual_test.py:135]: Yes, that would be it. Sorry for letting this file get so out of date.

Comment (MollieBakal) [reply test/manual_test.py]: Not a problem at all. Just committed with new name.

**Review by vringar — APPROVED** [review]
